### PR TITLE
LPS 134760 Page audit error handling

### PR DESCRIPTION
**Pull request by beltranrengifo**
[Initial PR](https://github.com/brianchandotcom/liferay-portal/pull/104131)

### CODE REVIEW

**Comment by beltranrengifo** [comment]
@4lejandrito Can you take a look please?

File: modules/apps/layout/layout-reports-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -70,11 +73,17 @@ link-text=Link Text
 no-site-exists-with-site-id-x=No site exists with the site ID {0}.
 overlapping-target=Overlapping Target
 page-audit=Page Audit
+page-not-accessible-from-internet=It is not accessible from internet or indexed. Private pages, local pages or some intranets can't be audited.
```
Comment (4lejandrito) [inline modules/apps/layout/layout-reports-web/src/main/resources/content/Language.properties:76]: can't -> cannot.

File: modules/apps/layout/layout-reports-web/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -70,11 +73,17 @@ link-text=Link Text
...
+set-api-key=Set API Key
+show-details=Show Details
 showing-data-from-x-relaunch-to-update-data=Showing data from {0}. Relaunch to update data.
 source-file=Source File
 strategy=Preferred Strategy
 strategy-description=Select what kind of device you want to audit for. Mobile strategy includes additional checks. Selected strategy will be applied to all page audits.
+the-api-key-is-invalid=The API key is invalid.
 there-are-no-x-related-issues=There are no {0} related issues.
+this-page-can't-be-audited=This page can't be audited.
```
Comment (4lejandrito) [inline modules/apps/layout/layout-reports-web/src/main/resources/content/Language.properties:85]: Same here.

**Comment by 4lejandrito** [comment]
Please @beltranrengifo set the title of the PR to the previous one so that it is easier to identify on the PRs list.

File: modules/apps/layout/layout-reports-web/src/main/resources/content/Language.properties (hunk 1 above)
Comment (beltranrengifo) [reply modules/apps/layout/layout-reports-web/src/main/resources/content/Language.properties]: Done

**Review by 4lejandrito — APPROVED** [review]

**Comment by beltranrengifo** [comment]
ci:forward

**Comment by beltranrengifo** [comment]
ci:forward

**Comment by beltranrengifo** [comment]
ci:forward

**Comment by beltranrengifo** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/104370.

**Comment by 4lejandrito** [comment]
ci:stop

**Pull request closed without merging**